tests/test_return_class.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from src.return_class import *
```

Comment:
```suggestion
from src.return_class import ReturnClass
```
It is better to import only what you use. Not everything. * means you taking everything from the class. If you change it to `ReturnClass` It will import only this